app/controllers/candidate_interface/other_qualifications/review_controller.rb

```diff
@@ -3,5 +3,19 @@ class OtherQualifications::ReviewController < CandidateInterfaceController
     def show
       @application_form = current_candidate.current_application
     end
+
+    def complete
+      @application_form = current_candidate.current_application
+
+      @application_form.update!(application_form_params)
```

Comment:
what if the updates go wrong, is there any validation for this?

Reply:
(I've just realised ... maybe this should not be a concern since it is a boolean)